pkg/apis/addtoscheme_serving_v1alpha1.go

```diff
@@ -2,9 +2,10 @@ package apis
 
 import (
 	"github.com/openshift-knative/knative-serving-operator/pkg/apis/serving/v1alpha1"
+	"k8s.io/kube-aggregator/pkg/apis/apiregistration/v1beta1"
 )
 
 func init() {
 	// Register the types with the Scheme so the components can map objects to GroupVersionKinds and back
-	AddToSchemes = append(AddToSchemes, v1alpha1.SchemeBuilder.AddToScheme)
+	AddToSchemes = append(AddToSchemes, v1alpha1.SchemeBuilder.AddToScheme, v1beta1.SchemeBuilder.AddToScheme)
```

Comment:
I feel like this should move to the  `Configure` function of the Openshift plugin instead. Is there a similar `Install` function available as we use there to install Openshift APIs?

Reply:
Yes, agree :+1:  Updated.